lib/index.coffee

```diff
@@ -4,10 +4,19 @@ GotoView = require('./goto-view')
 
 module.exports =
 
-  configDefaults:
-    logToConsole: false
-    moreIgnoredNames: ''
-    autoScroll: true
+  config:
+    logToConsole:
+      default: false
+      type: 'boolean'
+      description: 'Turn on logging debug information for goto commands'
+    moreIgnoredNames:
+      default: ''
+      type: 'string'
+      description: 'Whitespace-or-comma separated value of globs for filenames that goto will not generate symbols for. goto also ignores files in the core.ignoredNames setting.'
+    autoScroll:
+      default: true
+      type: 'boolean'
+      description: 'Disable to prevent returning jumping back to your original cursor position after cancelling a goto method'
```

Comment:
How about: `Disable this option to prevent goto from restoring your selection back to your original cursor position after cancelling a goto method`